### PR TITLE
add fields convert to lower case

### PR DESCRIPTION
**Pull request by perfectacle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,6 +174,13 @@ You can ignore these tables in the configuration like:
 skip_no_pk_table = true
 ```
 
+## Convert lower case for columns
```
Comment (siddontang) [inline README.md]: We don't need to explain every config field in the readme, please remove this.

File: etc/river.toml [hunk 2]
```diff
@@ -44,6 +44,9 @@ flush_bulk_time = "200ms"
 # Ignore table without primary key
 skip_no_pk_table = false
 
+# Set true when saving fileds to lower case
```
Comment (siddontang) [inline etc/river.toml]: True to use the lower-case fields.

File: etc/river.toml [hunk 3]
```diff
@@ -44,6 +44,9 @@ flush_bulk_time = "200ms"
...
+#to_lower_fields = true
```
Comment (siddontang) [inline etc/river.toml]: default is false.

File: river/config.go [hunk 4]
```diff
@@ -42,6 +42,11 @@ type Config struct {
 	FlushBulkTime TomlDuration `toml:"flush_bulk_time"`
 
 	SkipNoPkTable bool `toml:"skip_no_pk_table"`
+
+	ToLowerColumns bool `toml:"to_lower_fields"`
```
Comment (siddontang) [inline river/config.go]: ToLowerFields

File: river/config.go [hunk 5]
```diff
@@ -42,6 +42,11 @@ type Config struct {
...
+
+	// It contains converted table(eg, lower case fields)
+	ConvertedTable []string
```
Comment (siddontang) [inline river/config.go]: 1. If we have too many tables, the slice comparison will be very slow.
2. How do you handle DDL (we will support it later)

I don't think we need this field here, now the easiest way is to covert the field name every time (you may abstract a function to do this) when you set the key to the request data map in makeInsertReqData and makeUpdateReqData.

**Comment by siddontang** [comment]
PTAL @EagleChen

**Comment by EagleChen** [comment]
@perfectacle 

First, a simple `ToLowerFileds` is enough. If it's set, filed names are updated during syncing.

Second, actually I don't think there are many users who need this.
Both Mysql and Elasticsearch have related functions to convert fields to lower case.

**Comment by perfectacle** [comment]
I think we don't need this feature, so I close this pull request.

**Pull request closed without merging**